### ISSUE TITLE
Suggestion: option after install to copy UI prefabs/scenes even deeper

### ISSUE THREAD

**Issue by hsandt** [Maaack/Godot-Game-Template#74]
I really like the prefab copy (or rather, inheritance) system on start, but I still need to customize the option widgets further (there are too big on my lores pixel art game, I need to change some HBox to VBox, etc.).

Maybe we could have an option to create copies (or rather, inheriting scenes, since I see the copy inherit from the addon scenes, which is great) of low-level widgets as well?

Right now I must do it manually and remember to reassign references everywhere accordingly.

**Comment by Maaack** [comment]
Yeah, it definitely limits the flexibility. Maybe some of this could be accomplished by moving more scenes into `examples/` and inheriting from even simpler `base/` scenes, but I'm hesitant to do another refactor. I think an option for a deep copy makes the most sense right now, too.

**Comment by Maaack** [comment]
My latest recommendation for this case is to clear inheritance (right-click root node of scene) of any scenes a developer intends to customize. The scene inheritance helps for making the template, but I've only had it get in the way when developing a game. Clearing scene inheritance doesn't affect the class inheritance, either.
